ChangeLog

```diff
@@ -1,3 +1,9 @@
+Version 11.6.1
```

Comment:
This is going to conflict with https://github.com/thought-machine/please-servers/pull/269 when we land, but we should see that soon.

Reply:
Yeah I know. As said on that PR, I'll land it later in case it breaks something